src/das-employer-favourites.ApplicationServices/Domain/IFatRepository.cs

```diff
@@ -0,0 +1,9 @@
+using System.Threading.Tasks;
+
+namespace DfE.EmployerFavourites.ApplicationServices.Domain
+{
+    public interface IFatRepository
```

Comment:
I think this should live in Infrastructure. I don't think it's specifically a domain service. On it's own. I think the Implementation of the favourites domain service just uses a number of sources.